File: AULA_02/Haskell/NotaConceito.hs
conceito :: Bool -> Bool -> String
conceito _ True = "O"
conceito True False = "A"
conceito False False  = "R"

main = do
       putStrLn "Entre com a nota:  "
       line <- getLine
       let nota  = read line
       putStrLn "Entre com limiar: "
       line <- getLine
       let limiar  = read line
       putStrLn "Entre com as faltas: "
       line <- getLine
       let faltas  = read line
       putStrLn "Entre com max. faltas: "
       line <- getLine
       let maxFaltas  = read line
       let c = conceito ((nota :: Float) > (limiar :: Float))  ((faltas :: Int) > (maxFaltas :: Int))
       putStrLn c

File: AULA_02/C/NotaConceito.c
#include <stdio.h>

int main(){

  float nota, limiar;
  int faltas, maxFaltas;

  printf("Digite a nota e o limiar: ");
  scanf("%f %f",&nota,&limiar);
  printf("Digite as faltas e o limiar: ");
  scanf("%d %d",&faltas,&maxFaltas);

  if( faltas > maxFaltas ){
    printf("O\n");
  }else{
    if( nota > limiar ){
      printf("A\n");
    }else{
      printf("R\n");
    }
  }

  return 0;
}
